inspirehep/modules/theme/static/js/settings.js

```diff
@@ -55,6 +55,8 @@ require.config({
     'inspirehep-clipboard': 'js/inspire_clipboard',
     'impact-graphs': 'node_modules/impact-graphs/impact-graph',
     'inspirehep-search': 'node_modules/inspirehep-search-js/dist/inspirehep-search',
+    'inspire-profile-matcher-module': 'js/inspire_profile_matcher/profile_matcher/profilematcher.module',
+    'inspire-profile-matcher-controllers': 'js/inspire_profile_matcher/profile_matcher/profilematcher.controller',
```

Comment:
It doesn't load the controller without it. Can I declare the dependency somewhere else?